lib/capistrano/tasks/shared_configs.rake

```diff
@@ -3,6 +3,17 @@ def repo_config_path
 end
 
 namespace :shared_configs do
+  desc 'Checks the project is ready to work with the shared configs directory'
+  task :check do
+    on roles(:app) do
+      if test("[ -d #{repo_config_path} ]")
+        puts "Found shared configs in #{repo_config_path}."
+      else
+        puts "Unable to locate shared configs in #{repo_config_path}."
+      end
+    end
+  end
+
```

Comment:
Note that no other tasks call this or depend on it, at this stage of the PR.  Maybe they should?